client/service/src/config.rs

```diff
@@ -87,6 +87,8 @@ pub struct Configuration {
 	/// External WASM transport for the telemetry. If `Some`, when connection to a telemetry
 	/// endpoint, this transport will be tried in priority before all others.
 	pub telemetry_external_transport: Option<ExtTransport>,
+	/// Telemetry senders
+	pub telemetry_senders: sc_telemetry::Senders,
```

Comment:
What are these senders?

Reply:
They are channel's senders to send messages to the telemetry workers. It needs to be "globalized" so I can register new telemetry instances (the `Senders` is used in the `Layer` [here](https://github.com/paritytech/substrate/blob/389da318cf94d8a47e824c0766675cb6ad279ec4/client/telemetry/src/layer.rs#L29))

When I will implement the one-common-connection-for-multiple-nodes feature (in this PR soon) I will probably move around a "Telemetries" object which will also hold the workers so they can be re-used by other nodes.